models/errors.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+// SPVError is extended error which holds information about http status and code that should be returned
+type SPVError struct {
+	Code       string
+	Message    string
+	StatusCode int
+}
+
+// ResponseError is an error which will be returned in HTTP response
+type ResponseError struct {
+	Code    string `json:"Code"`
+	Message string `json:"Message"`
+}
+
+const UnknownErrorCode = "error-unknown"
```

Comment:
comment 
```
// UnknownErrorCode is the default error code for unknown errors
```

also not sure if just "unknown" would not be enough in the message, what u think ?

Reply:
`error-` prefix corresponds to the structure of all error